src/applications/personalization/profile-2/components/AccountSecurity.jsx

```diff
@@ -27,7 +29,7 @@ class AccountSecurity extends Component {
           Account security
         </h2>
         <DowntimeNotification
-          appTitle="Account Security"
+          render={handleDowntimeForSection('account security')}
```

Comment:
This overrides the default messaging, making it consistent across all of the Profile 2.0 sections